collections/oneruntime/general-notifications/event-general-notifications/account-balance-changed/event-general-notifications.json

```diff
@@ -9,9 +9,8 @@
   "generalNotifications": [
     {
       "id": "account-balance-low",
-      "name": "Account Balance Low Retail",
+      "name": "Account Balance Low - Retail",
       "userPreferenceEnabled": true,
-      "manageable": false,
```

Comment:
isn't it better to keep it as true/false in all GNs instead if removing since some of them still have it even if it is false 😅

Reply:
hey @afsinka-backbase we can discuss it. Is expected to add a flag for default behavior? I removed it as false and didn't add as true, because true is the default value

Reply:
matter of taste, up to you